sdk/mixedreality/Microsoft.Azure.Management.MixedReality/src/Properties/AssemblyInfo.cs

```diff
@@ -22,5 +22,5 @@
 [assembly: ComVisible(false)]
 
 // The following GUID is for the ID of the typelib if this project is exposed to COM
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.1.0.0")]
```

Comment:
The assembly version should match package version. Is there a reason it is being kept at 1.xxx?

Reply:
We leave this at 1.0.0.0 and it will get updated automatically?

Reply:
I don't think it get updated automatically, but I'd wait for @allenjzhang confirmation.

The package version as is stated in the [.csproj file](https://github.com/Azure/azure-sdk-for-net/pull/18869/files#diff-893c8a648c1d0471282af62e5b78e6b68cbfdef3f5c16a4f70efcba8312716f5) right now is `3.0.0`, which doesn't match the assembly version here.

Reply:
Ok, I look at other services they are also in 1.x.x.x I can do the change to 3 if we want that.

Reply:
If you look at other SDKs, the version in the Assembly/FileVersion matches the version in the .csproj.

Reply:
Ok I will make them match then. Thanks!